docs/pages/versions/unversioned/sdk/updates.mdx

```diff
@@ -10,102 +10,126 @@ platforms: ['android', 'ios', 'tvos']
+The library also provides a variety of constants to inspect the current update and functions to customize update behavior from your application code (after startup). For example, one common alternative usage pattern is to manually check for updates after the app has started instead of doing the default check on launch.
 
-On iOS, these properties are set as keys in the **Expo.plist** file. You can also set them at runtime by calling `[EXUpdatesAppController overrideConfigurationWithConfiguration:]` at any point before `expo-updates` is initialized, and the values in this dictionary will override any values specified in **Expo.plist**. If you tried to use the `EXUpdatesAppController` from the **AppDelegate.mm**, you will need to add the following imports:
+<Collapsible summary="Example: Check for updates manually">
 
-```objc From Objective-C to import Swift headers for expo-updates
-#import "ExpoModulesCore-Swift.h"
-#import "EXUpdatesInterface-Swift.h"
-#import "EXUpdates-Swift.h"
-```
+You can configure your app to check for updates manually by doing the following steps:
+1. Set the `checkAutomatically` configuration value to `ON_ERROR_RECOVERY` or `NEVER` to disable the library's default launch behavior.
+2. Add the following code to check for available updates, download them, and reload:
+
+    ```jsx App.js
+    import { View, Button } from 'react-native';
+    import * as Updates from 'expo-updates';
+
+    function App() {
+      async function onFetchUpdateAsync() {
+        try {
+          const update = await Updates.checkForUpdateAsync();
+
+          if (update.isAvailable) {
+            await Updates.fetchUpdateAsync();
+            await Updates.reloadAsync();
+          }
+        } catch (error) {
+          // You can also add an alert() to see the error message in case of an error when fetching updates.
+          alert(`Error fetching latest Expo update: ${error}`);
+        }
+      }
+
+      return (
+        <View>
+          <Button title="Fetch update" onPress={onFetchUpdateAsync} />
+        </View>
+      );
+    }
+    ```
+
+</Collapsible>
+
+## Testing
+
+Most of the methods and constants in this library can be used or tested only in release builds. In debug builds, the default behavior is to always load the latest JavaScript from a development server. It is possible to [build a debug version of your app with the same updates behavior as a release build](/eas-update/debug-advanced/#debugging-of-native-code-while-loading-the-app-through-expo-updates). Such an app will not open the latest JavaScript from your development server &mdash; it will load published updates just as a release build does. This may be useful for debugging the behavior of your app when it is not connected to a development server.
```

Comment:
```suggestion
Most of the methods and constants in this package can be used or tested only in release builds. In debug builds, the default behavior is to always load the latest JavaScript from a development server. It is possible to [build a debug version of your app with the same updates behavior as a release build](/eas-update/debug-advanced/#debugging-of-native-code-while-loading-the-app-through-expo-updates). Such an app will not open the latest JavaScript from your development server &mdash; it will load published updates just as a release build does. This may be useful for debugging the behavior of your app when it is not connected to a development server.
```